lms/djangoapps/certificates/api.py

```diff
@@ -116,7 +125,7 @@ def generate_user_certificates(student, course_key, course=None, insecure=False,
         generate_pdf=generate_pdf,
         forced_grade=forced_grade
     )
-    if cert.status in [CertificateStatuses.generating, CertificateStatuses.downloadable]:
+    if is_passing_status(cert.status):
```

Comment:
Why not use `CertificateStatuses.is_passing_status(cert.status)` here and avoid method above?